tests/operator/helper.go

```diff
@@ -45,3 +52,95 @@ func SplitCsv(csv string) CsvResult {
 	}
 	return result
 }
+
+func getAllPodsBy(namespace string) (podsInNamespace []*provider.Pod) {
+	for i := range env.AllPods {
```

Comment:
Just in general I feel like these filtering functions should accept `env.AllPods` as a parameter instead of just referencing the global variable.  Not a huge deal but for unit testing functions it makes it easier.

Reply:
Makes sense.